Gaijinizer patch/Map537.txt

```diff
@@ -4,243 +4,310 @@
+of singularity.#Ron
 > END STRING
 
 > BEGIN STRING
 ボクには無関係な世界だから、といえばそこまでだけど
 
 辛いだろうなぁ……ルカくん。
-> CONTEXT: Map537/events/7/pages/0/28/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/28/Dialogue/enemy29
+That's all I can say since that world has nothing
+to do with me.
 
+But it must be rough, Luka-kun...#Ron
 > END STRING
 
 > BEGIN STRING
 ………！！
 この感じ……\\C[13]『白兎』\\C[0]…！？
 
 い、いや、そんなわけない……！！
-> CONTEXT: Map537/events/7/pages/0/53/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/53/Dialogue/enemy29
+......!!
 
+That sensation... \\C[13]White Rabbit\\C[0]?!
+
+N-No, it can't be!!#Ron
 > END STRING
 
 > BEGIN STRING
 でもこの感じは……
 
 もう、すぐ近くにいる……！
-> CONTEXT: Map537/events/7/pages/0/58/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map537/events/7/pages/0/58/Dialogue/enemy29
+But that feeling's already so close!#Ron
 > END STRING
 
 > BEGIN STRING
 ルカくん……その姿……！！
 
 そうか……\\C[2]やっぱり気休めにもならなかったか……
-> CONTEXT: Map537/events/7/pages/0/119/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/119/Dialogue/enemy29
+Luka-kun... that form!!
 
+I see... \\C[2]so it really didn't do anything...#Ron
 > END STRING
 
 > BEGIN STRING
 ルカらしき人物は、チェシャに近付いてきた……
 
 その眼には、明らかな\\C[10]殺気\\C[0]が込められている……
-> CONTEXT: Map537/events/7/pages/0/126/Dialogue < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/126/Dialogue
+The individual resembling Luka has approached Cheshire...
 
+There's a distinct \\C[10]desire to kill \\C[0]manifest in its eyes...#Ron
 > END STRING
 
 > BEGIN STRING
 ………………もう、理性も消えちゃったんだね。
 
 ボクの気配だけを頼りに、ここまで辿り着いたのか……
-> CONTEXT: Map537/events/7/pages/0/130/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/130/Dialogue/enemy29
+......You've already lost all reason.
 
+You relied on following my presence to get here.#Ron
 > END STRING
 
 > BEGIN STRING
 さて……どうしたものかな。
 
 ボクとしては、大きな問題が一段落した直後だから
 もうこれ以上の揉め事はイヤなんだけど……
-> CONTEXT: Map537/events/7/pages/0/137/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/137/Dialogue/enemy29
+Well then, what to do.
 
+I've only just resolved one huge problem so I
+don't feel like getting into any more scraps.#Ron
 > END STRING
 
 > BEGIN STRING
 キミがどうしてもっていうなら……
 
 相手をしてあげないといけないのかな……
-> CONTEXT: Map537/events/7/pages/0/142/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map537/events/7/pages/0/142/Dialogue/enemy29
+But if you really want to fight me no matter
+what, I suppose I'll take you on.#Ron
 > END STRING
 
 > BEGIN STRING
 ？？？「対象、発見」
-> CONTEXT: Map537/events/7/pages/0/150/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map537/events/7/pages/0/150/Dialogue
+??? 「Target... confirmed...」#Ron
 > END STRING
 
 > BEGIN STRING
 第零種断界接触、および第二法則侵蝕率0.986……
 イレギュラー個体を処断する。
-> CONTEXT: Map537/events/7/pages/0/161/Dialogue/kureo_adora_fc1 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/161/Dialogue/kureo_adora_fc1
+Class Zero Cross-World Contact detected...
+Second Law Corruption Rate of 0.986%...
 
+Passing sentence on irregularity...#Ron
```

Comment:
The corruption rate looks to be 98.6%, not 0.986%, especially given the so-commented pinch for the Paradox world.
With the last line I think it would make more sense to say "executing irregularity".